config/dbConn.js

```diff
@@ -0,0 +1,13 @@
+import mongoose from 'mongoose';
+
+const connectDB = async () => {
+  try {
+    await mongoose.connect(
+      process.env.DATABASE_URI, 
+    );
+  } catch (err) {
+    console.error(err)
+  }
+}
```

Comment:
this just sets up our connection to MongoDB using mongoose- we use it in index.js when booting up the server

Reply:
from an outdated comment:
This is how you connect to MongoDB- it's like a cloud instance of a DB. Mongoose is the package that acts like a wrapper/ORM/interface with MongoDB for Node.js